rootfs/etc/confd/templates/builder

```diff
@@ -64,17 +64,30 @@ TMP_DIR=$(mktemp -d -p $BUILD_DIR)
 
 cd $REPO_DIR
 # use Procfile if provided, otherwise try default process types from ./release
+
+git archive --format=tar.gz ${GIT_SHA} > ${APP_NAME}.tar.gz
```

Comment:
also from the changes here I don't see any technical reason why we need to retain a second copy of the application files in $TMP_DIR.